input/fsh/CodeSystem-Mondo.fsh

```diff
@@ -0,0 +1,13 @@
+CodeSystem: Mondo
+Id: mondo
+Title: "Mondo Disease Ontology"
+Description: "The Mondo Disease Ontology (Mondo) aims to harmonize disease definitions across the world. The name Mondo comes from the latin word 'mundus' and means 'for the world.'"
```

Comment:
`The Mondo Disease Ontology (Mondo)` >> `The Mondo Disease Ontology (MONDO)`

Reply:
That's how they spell it here: https://mondo.monarchinitiative.org/ (it's not a complete acronym)

Reply:
Ignore!